tools/automator/automator.sh

```diff
@@ -229,13 +229,13 @@ work() { (
 
   AUTOMATOR_REPO_DIR="$(pwd)"
 
-  bash "${shell_args[@]}" "$script_path" "${script_args[@]}" || print_error "unable to execute command for: $repo"
```

Comment:
Do we not think this is useful debugging information? Updating this function to do:

```bash
print_error() {
  ret=$?
  echo "$@" >&2
  return $ret
}

bash $shell_args $script_path $script_args || print_error "unable to execute command for $repo"
```

Will give you the desired behavior and preserve the debugging.


On the other hand "unable to execute command" is a poorly formed message (tell me the command, please)

I'd favor doing something more like this:
```bash
log-command() {
  set -o xtrace 
  $@
}

log-command bash $shell $script $args
```

Reply:
Thanks, I think the trace would be beneficial here. I will update.

Reply:
Wait, I just realized that I already added support for xtrace for the command/script via the `--verbose` flag.

Reply:
Added a `trace` function.